apps/web/modules/users/views/users-public-view.getServerSideProps.tsx

```diff
@@ -206,7 +207,10 @@ export const getServerSideProps: GetServerSideProps<UserPageProps> = async (cont
 
   const profile = {
     name: user.name || user.username || "",
-    image: user.avatar,
+    image: getUserAvatarUrl({
+      ...user,
+      profile: user.profile,
+    }),
```

Comment:
This was returning `"image": "/peer/avatar.png"` which is wrong.

<img width="533" alt="image" src="https://github.com/calcom/cal.com/assets/3504472/209d54f8-7a00-4d32-9b0f-cc213b85c8a5">